### ISSUE TITLE
Can't install das due to tensorflow version conflicts 

### ISSUE THREAD

**Issue by verity-cook** [janclemenslab/das#63]
When trying to install das following the instructions:

```
conda install mamba -c conda-forge -n base -y
mamba create python=3.9 das=0.31.0 -c conda-forge -c ncb -c anaconda -c nvidia -n das -y
```

I get the following error:

Could not solve for environment specs
The following package could not be installed
└─ das 0.31.0**  is uninstallable because it requires
   └─ tensorflow-gpu >2.3  but there are no viable options
      ├─ tensorflow-gpu 2.5.0 would require
      │  └─ tensorflow 2.5.0 , which conflicts with any installable versions previously reported;
      ├─ tensorflow-gpu 2.6.0 would require
      │  └─ tensorflow 2.6.0 , which conflicts with any installable versions previously reported;
      └─ tensorflow-gpu [2.5.0|2.6.0] conflicts with any installable versions previously reported.

**Comment by postpop** [comment]
Hi, which system are you on?

**Comment by verity-cook** [comment]
Windows

**Comment by postpop** [comment]
Will look into it - tensorflow is a constant source of issues. thanks for raising the issue!

**Comment by verity-cook** [comment]
Thank you! And thanks for replying so promptly

**Comment by postpop** [comment]
Hi, I've rebuilt the package and it passed the installation test. Can you give it another try?

**Comment by verity-cook** [comment]
I get the same error when I run this line:
`mamba create python=3.9 das=0.31.0 -c conda-forge -c ncb -c anaconda -c nvidia -n das -y`

Does the mamba version matter? I'm using mamba 1.4.2

**Comment by kaede-kmr** [comment]
I've got similar issues during installation on Mac (M1 Macbook Air).

Code executed:
`mamba create python=3.9 das=0.31.0 -c conda-forge -c ncb -c anaconda -c nvidia -n das -y`

Result:
```
Could not solve for environment specs
The following package could not be installed
└─ das 0.31.0**  is not installable because it requires
   └─ tensorflow-deps 2.9.* , which does not exist (perhaps a missing channel).
```

I also tried to install tensorflow-deps 2.9.0 independently from the apple channel. 
`mamba install tensorflow-deps=2.9.0 -c apple`

However, this raises a much longer dependency-conflict list.
Should I install tensorflow-deps in different ways?
